application/backend/src/main/java/io/typerefinery/websight/models/components/BarChart.java

```diff
@@ -18,6 +18,6 @@ public class BarChart {
     
     @Getter
     @Inject
-    // @Default (values = "http://localhost:8080/apps/typerefinery/components/content/lineChart/dataSource_1.json")
+    // @Default (values = "http://localhost:8080/apps/typerefinery/components/content/barChart/dataSource_1.json")
```

Comment:
what are these comments for?

Reply:
It is for default json for line chart component.